qiita_ware/ebi.py

```diff
@@ -47,39 +48,7 @@ class NoXMLError(Exception):
 def clean_whitespace(s):
     """Standardizes whitespace so that there is only ever one space separating
```

Comment:
I know you didn't modify this, but can you reword this sentence so it fits in a single line in order to follow the numpy standard?